src/main/java/carRacing/CarRacingMain.java

```diff
@@ -0,0 +1,35 @@
+package carRacing;
+
+import carRacing.car.Car;
+import carRacing.car.CarRacing;
+import carRacing.car.CarWinner;
+import carRacing.numberMaker.NumberMaker;
+import carRacing.numberMaker.RandomNumberMaker;
+import carRacing.view.InputView;
+import carRacing.view.ResultView;
+import java.util.List;
+
+public class CarRacingMain {
+
+	public static void main(String[] args) {
+		InputView input = new InputView();
+		String[] nameInputs = input.inputCarName();
+		int numberOfAttempts = input.inputMoveCount();
+
+		System.out.println("실행 결과");
+
+		ResultView result = new ResultView();
+		CarRacing carRacing = new CarRacing();
+		List<Car> cars = carRacing.makeCars(nameInputs.length, nameInputs);
+		NumberMaker numberMaker = new RandomNumberMaker();
+
+
+		for (int i = 0; i < numberOfAttempts; i++) {
+			carRacing.moveCars(numberMaker);
+			result.showResult(cars);
+		}
```

Comment:
```suggestion
		CarRacing carRacing = new CarRacing(names);
		NumberMaker numberMaker = new RandomNumberMaker();

		for (int i = 0; i < numberOfAttempts; i++) {
			carRacing.moveCars(numberMaker);
			result.showResult(carRacing.getCars());
		}
```

CarRacing을 이렇게 활용해 볼 수 있겠네요